backend/core/Store.qml

```diff
@@ -22,6 +23,14 @@ Item {
         {
             name: qsTr("Compliance Report"),
             page: "../pages/CompliancePage.qml"
+        },
+        {
+            name: "Project / Deploy",
```

Comment:
Should use qsTr and be renamed something like Projects Continuous Deployment (whatever the name, it should mention continuous deployment as this is the purpose of this new page for now).